src/python/ensembl/production/datafile/file_serializers.py

```diff
@@ -0,0 +1,33 @@
+#  See the NOTICE file distributed with this work for additional information
```

Comment:
I would call this file only `serializers.py`

Reply:
That's a bit too generic IMO considered the shared `ensembl.production.datafile` namespace where other packages (at the moment only `datafile-toolchain`) will potentially have their modules.

Reply:
See refactored layout